Sources/SwiftFormatPrettyPrint/TokenStreamCreator.swift

```diff
@@ -981,6 +981,14 @@ private final class TokenStreamCreator: SyntaxVisitor {
   }
 
   func visit(_ node: MemberDeclListItemSyntax) -> SyntaxVisitorContinueKind {
+    if shouldFormatterIgnore(node: node) {
+      appendFormatterIgnored(node: node)
+
+      // Add this break so that trivia parsing will allow discretionary newlines after the node.
+      appendToken(.break(.same, size: 0))
```

Comment:
Can this be moved into `appendFormatterIgnored(node:)` to reduce a little bit of the call sites' duplication?

Reply:
Good call. Done.